src/components/RepositoryMetadata/RepositoryMetadata.tsx

```diff
@@ -76,13 +76,13 @@ export const RepositoryMetadata: React.FunctionComponent<Props> = ({
           <>
             { repo.re3dataId && (
               <div>
-                <a href={re3DataURL()}>
+                <a id="re3data-link" href={re3DataURL()}>
```

Comment:
I think we need to make these more unique.

Since this is called in a loop for each repository, the `a` tags cannot all have the same ID.